revocation/internal/crl/crl.go

```diff
@@ -197,9 +201,7 @@ func Supported(cert *x509.Certificate) bool {
 }
 
 func hasFreshestCRL(extensions []pkix.Extension) bool {
```

Comment:
This one-line helper function becomes redundant.

Reply:
Updated.